lib/ui/contact_list/new_chat_bottom_sheet.dart

```diff
@@ -383,10 +382,14 @@ class _NewChatBottomSheetState extends ConsumerState<NewChatBottomSheet> {
           focusNode: _searchFocusNode,
           hintText: 'Search contact or public key...',
           decoration: InputDecoration(
-            prefixIcon: Icon(
-              CarbonIcons.search,
-              color: context.colors.primary,
-              size: 20.sp,
+            prefixIcon: SvgPicture.asset(
+              AssetsPaths.icSearch,
```

Comment:
Probably related to the previosu comment cause it is the same icon. Here the issue is only size, not color. This is how it looks
|Light mode| Dark mode|
|----|----|
|<img width="363" height="758" alt="Screenshot 2025-08-16 at 12 00 26 AM" src="https://github.com/user-attachments/assets/3deaf6ed-0e81-4e11-91bc-425f547e6ab4" />|<img width="371" height="762" alt="Screenshot 2025-08-16 at 12 01 28 AM" src="https://github.com/user-attachments/assets/a53436df-4b88-41c8-9a5f-442f3fc1bf72" />|